homeassistant/components/lovelace/__init__.py

```diff
@@ -334,6 +352,41 @@ def delete_card(fname: str, card_id: str, position: int = None):
         "Card with ID: {} was not found in {}.".format(card_id, fname))
 
 
+def get_view(fname: str, view_id: str, data_format: str = FORMAT_YAML):
+    """Get view without it's cards."""
+    config = load_yaml(fname)
+    view = next(
```

Comment:
let's use a `for` loop with `break` for readability. Also, you're now calling `str` on `None` which is weird